fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/FabricItem.java

```diff
@@ -119,15 +123,27 @@ default ItemStack getRecipeRemainder(ItemStack stack) {
 	 * <p>Note that this method is only called <em>after</em> the {@link EnchantmentEvents#ALLOW_ENCHANTING} event, and
 	 * only if none of the listeners to that event override the result.</p>
 	 *
-	 * @param stack the current stack
+	 * @param stack       the current stack
 	 * @param enchantment the enchantment to check
-	 * @param context the context in which the enchantment is being checked
+	 * @param context     the context in which the enchantment is being checked
 	 * @return whether the enchantment is allowed to apply to the stack
 	 */
 	default boolean canBeEnchantedWith(ItemStack stack, Enchantment enchantment, EnchantingContext context) {
 		return enchantment.isAcceptableItem(stack);
 	}
 
+	/**
+	 * Retrieves the normal 'lifespan' of this item when it is dropped on the ground
+	 * as an ItemEntity. This is in ticks, standard result is 6000, or 5 mins.
+	 *
+	 * @param stack The current ItemStack
+	 * @param world The level the entity is in
+	 * @return The normal lifespan in ticks.
```

Comment:
```suggestion
	 * @return the normal lifespan in ticks
```
This formatting should be consistent within the class.